Client/Frontend/Browser/Playlist/Controllers/PlaylistMoveFolderView.swift

```diff
@@ -0,0 +1,269 @@
+                .aspectRatio(contentMode: .fit)
+                .foregroundColor(iconColor)
+            
+            VStack(alignment: .leading) {
+                if let folder = folder {
+                    let itemCount = folder.playlistItems?.count ?? 0
+                    
+                    Text(folder.title ?? "")
+                        .font(.body)
+                        .foregroundColor(titleColor)
+                    Text("\(itemCount) \(itemCount == 1 ? Strings.PlaylistFolders.playlistFolderSubtitleItemSingleCount : Strings.PlaylistFolders.playlistFolderSubtitleItemCount)")
+                        .font(.footnote)
+                        .foregroundColor(titleColor)
+                } else {
+                    Spacer()
+                }
+            }
+            
+            if selectedFolder?.uuid == folder?.uuid {
+                Spacer()
+                Image(systemName: "checkmark")
+                    .renderingMode(.template)
+                    .aspectRatio(contentMode: .fit)
+                    .foregroundColor(.white)
+            }
+        }
+        .padding(EdgeInsets(top: 7.0, leading: 0.0, bottom: 7.0, trailing: 0.0))
+    }
+}
+
+struct PlaylistMoveFolderView: View {
+    @FetchRequest(
+        entity: PlaylistFolder.entity(),
+        sortDescriptors: [
+            NSSortDescriptor(keyPath: \PlaylistFolder.order, ascending: true),
+            NSSortDescriptor(keyPath: \PlaylistFolder.dateAdded, ascending: false)
+        ],
+        predicate: NSPredicate(format: "uuid != %@", PlaylistFolder.savedFolderUUID)
+    ) var folders: FetchedResults<PlaylistFolder>
+    
+    @FetchRequest(
+        entity: PlaylistFolder.entity(),
+        sortDescriptors: [],
+        predicate: NSPredicate(format: "uuid == %@", PlaylistFolder.savedFolderUUID)
+    ) var savedFolder: FetchedResults<PlaylistFolder>
+    
+    @State private var moveDisabled: Bool = true
+    @State private var selectedFolder = PlaylistManager.shared.currentFolder
+    
+    var selectedItems: [PlaylistItem]
+    var onCancelButtonPressed: (() -> Void)?
+    var onDoneButtonPressed: (([PlaylistItem], PlaylistFolder?) -> Void)?
+    
+    var body: some View {
+        NavigationView {
+            Form {
+                Section(header: {
+                    HStack(spacing: 18.0) {
+                        if selectedItems.count > 1,
+                           let firstItem = selectedItems[safe: 0],
+                           let secondItem = selectedItems[safe: 1] {
+                            ZStack(alignment: .topLeading) {
+                                PlaylistFolderImage(item: firstItem)
+                                    .rotationEffect(.degrees(5.0))
+                                PlaylistFolderImage(item: secondItem).rotationEffect(.degrees(-5.0))
+                            }
+                            
+                            Text("\(firstItem.name ?? "") & \(selectedItems.count - 1) item(s)")
```

Comment:
Not localized, and remember the count should be included in the localization as mentioned above